pkg/proxy/metrics/metrics.go

```diff
@@ -113,6 +113,26 @@ var (
 			Help:      "Cumulative proxy rules Service changes",
 		},
 	)
+
+	// NodeChangesPending is the number of pending node changes that
+	// have not yet been synced to the proxy.
+	NodeChangesPending = prometheus.NewGauge(
```

Comment:
If we can eliminate the node-complexity, we should not need this, right?

Reply:
Yeah, I'll remove it.